src/NumberGuessingGame.js

```diff
@@ -1,26 +1,26 @@
-import React, { Component, useState } from "react";
+import React from "react";
 import GuessControl from "./GuessControl";
 import GuessMessage from "./GuessMessage";
 import GameOver from "./GameOver";
 
 function NumberGuessingGame() {
-  const [numberToGuess, setnumberToGuess] = React.useState("");
-  const [numberOfGuesses, setnumberOfGuesses] = React.useState("");
+  const [numberToGuess, setnumberToGuess] = React.useState(getRandomNumber());
+  const [numberOfGuesses, setnumberOfGuesses] = React.useState(0);
   const [latestGuess, setlatestGuess] = React.useState("");
 
-  function handleGuess(event) {
-    setlatestGuess(parseInt(latestGuess));
-    setnumberOfGuesses(numberOfGuesses +1);
+  function handleGuess(guess) {
+    setlatestGuess(guess);
+    setnumberOfGuesses(parseInt(numberOfGuesses) + 1);
   }
   function handleReset() {
     const resetAll = {
       numberToGuess: getRandomNumber(),
       numberOfGuesses: 0,
       latestGuess: null,
     };
-    setnumberToGuess(resetAll);
-    setnumberOfGuesses(resetAll);
-    setlatestGuess(resetAll);
+    setnumberToGuess(resetAll.numberToGuess);
```

Comment:
we aren't using the values of `resetAll` outside of these one time uses so do we even need `resetAll`? How can we make this look better?